spe1_brine/SPE1CASE1_BRINE_THERMAL.DATA

```diff
@@ -0,0 +1,390 @@
+  
+-- Oil/Water Viscosity as a function of temperature
+VISCREF
+ 300 0.93 /
+
+WATVISCT
+30	1.2
+40	1.06
+50	0.92
+60	0.78
+70	0.64
+80	0.5
+/
+
+SWOF
+
+--	Sw	Krw	Kro	Pc						
+	0.000000	0.000000	1.00000		3.75633	
+	0.0001		0.000000	0.999		3.75632						
+	0.0500000	0.000860000	0.847820	1.86981						
+	0.100000	0.00263000	0.697460	1.23731						
+	0.150000	0.00524000	0.557170	0.91821						
+	0.200000	0.00877000	0.432860	0.72451						
+	0.250000	0.0133800	0.327570	0.59341						
+	0.300000	0.0192700	0.241770	0.49811
+	0.350000	0.0267200	0.174150	0.42511
+	0.400000	0.0360800	0.122370	0.36691
+	0.450000	0.0478100	0.0837400	0.31911
+	0.500000	0.0625000	0.0556500	0.27881
+	0.550000	0.0809000	0.0357200	0.24401
+	0.600000	0.103940	0.0219900	0.21351
+	0.650000	0.132770	0.0128400	0.18631
+	0.700000	0.168690	0.00699000	0.16161
+	0.750000	0.213020	0.00346000	0.13901
+	0.800000	0.266670	0.00149000	0.11801
+	0.850000	0.329180	0.000510000	0.09831
+	0.900000	0.397060	0.000120000	0.07961
+	0.950000	0.461030	0.00001		0.06161
+	1.00000		0.500000	0.000000	0.04408 /
+/
+SGOF
+
+--  Sg	Krg	Kro	Pc	
+	0.000000	0.000000	1.00000		0.000000
+	0.0500000	0.00165500	0.806888	0.000000
+	0.100000	0.00691300	0.633562	0.000000
+	0.150000	0.0162130	0.485506	0.000000
+	0.200000	0.0299900	0.364043	0.000000
+	0.250000	0.0486550	0.267589	0.000000
+	0.300000	0.0725730	0.192992	0.000000
+	0.350000	0.102046	0.136554	0.000000
+	0.400000	0.137287	0.0946710	0.000000
+	0.450000	0.178402	0.0641510	0.000000
+	0.500000	0.225368	0.0423240	0.000000
+	0.550000	0.278030	0.0270350	0.000000
+	0.600000	0.336093	0.0165860	0.000000
+	0.650000	0.399135	0.00966200	0.000000
+	0.700000	0.466631	0.00525400	0.000000
+	0.750000	0.538000	0.00259700	0.000000
+	0.800000	0.612665	0.00111700	0.000000
+	0.850000	0.690169	0.000384000	0.000000
+	0.900000	0.770395	0.000088	0.000000
+	0.950000	0.854218	0.000007	0.000000
+ 	0.9999		0.9499		0.000000	0.000000	
+	1.00000		0.950000	0.000000	0.000000 /
+
+/
+
+SOLUTION
+-- -------------------------------------------------------------------------
+
+EQUIL
+-- Item 1: datum depth (ft)
+-- Item 2: pressure at datum depth (psia)
+-- 	   - Odeh's table 1 says that initial reservoir pressure is 
+-- 	     4800 psi at 8400ft, which explains choice of item 1 and 2
+-- Item 3: depth of water-oil contact (ft)
+-- 	   - chosen to be directly under the reservoir
+-- Item 4: oil-water capillary pressure at the water oil contact (psi)
+-- 	   - given to be 0 in Odeh's paper
+-- Item 5: depth of gas-oil contact (ft)
+-- 	   - chosen to be directly above the reservoir
+-- Item 6: gas-oil capillary pressure at gas-oil contact (psi)
+-- 	   - given to be 0 in Odeh's paper
+-- Item 7: RSVD-table
+-- Item 8: RVVD-table
+-- Item 9: Set to 0 as this is the only value supported by OPM
+
+-- Item #: 1 2    3    4 5    6 7 8 9
+    2560.32 331 2575.56 0 2560.32 0 1 0 0 /
+
+SALTVD
+ 2529.84  9
+ 2575.56  9 /
+
+RSVD
+-- Dissolved GOR is initially constant with depth through the reservoir.
```

Comment:
Tiny nit: This table has an initial dissolved GOR that varies with the depth so the comment does not agree with the data. Maybe you could fix that before we merge this?